### PR TITLE
Remove Mozilla-centric things (#233)

### PR DESCRIPTION
**Pull request by willkg**

### CODE REVIEW

**Comment by willkg** [comment]
@lonnen I think this covers everything we should do before transferring. Can you think of anything else?

File: CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Bleach was originally written and maintained by Erik Rose and various
```
Comment (hoodmane) [inline CONTRIBUTORS:1]:
```suggestion
sphinx-js was originally written and maintained by Erik Rose and various
```

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+contributors within and without the Mozilla Corporation and Foundation.
+
+It is currently maintained by XYZ.
```
Comment (hoodmane) [inline CONTRIBUTORS:4]:
```suggestion
It is currently maintained by Hood Chatham.
```
Or I can insert this myself later. Thanks!

File: CONTRIBUTORS (hunk 1 above)
Comment (willkg) [reply CONTRIBUTORS]: Thank you!

File: CONTRIBUTORS (hunk 2 above)
Comment (willkg) [reply CONTRIBUTORS]: Done!

**Review by lonnen — COMMENTED** [review]
If we're removing `CODE_OF_CONDUCT.md` we should also remove it from `MANIFEST.in`

**Comment by lonnen** [comment]
oops, this was already merged. I'm on it

**Comment by hoodmane** [comment]
I'm planning to add the code of conduct from Pyodide so it's probably not important. But also do we need to put the code of conduct into the wheel?

**Pull request merged**